ts-sdk/whirlpool/src/createPool.ts

```diff
@@ -60,17 +60,18 @@ export type CreatePoolInstructions = {
  *
  * @example
  * import { createSplashPoolInstructions } from '@orca-so/whirlpools';
- * import { generateKeyPairSigner, createSolanaRpc, devnet } from '@solana/web3.js';
+ * import { createKeyPairSignerFromBytes, createSolanaRpc, devnet, lamports } from '@solana/web3.js';
  *
  * const devnetRpc = createSolanaRpc(devnet('https://api.devnet.solana.com'));
- * const wallet = await generateKeyPairSigner();
+ * const keyPairBytes = new Uint8Array(JSON.parse(fs.readFileSync('path/to/solana-keypair.json', 'utf8')));
```

Comment:
On the one hand I like this because it is better practice than generatekeypair. On the other hand now each example requires a setup step. 

I'm in favor of leaving this as generatekeypair but wouldn't be opposed to adding a comment: "see wallat management on how to load a wallet" or something

Reply:
On the other hand don't really care as much as long as it is clear to the user

Reply:
Links to documention inside an example code-block does not look nice.
![image](https://github.com/user-attachments/assets/2ffed59a-04c9-474c-9e22-d5f0e6f81c57)

I will change to:
```tsx
const wallet = await generateKeyPairSigner(); // CAUTION: this wallet is no persistent.
```
Since the functions only generate instructions, and there's no mentioning of "SOL airdrop on Devnet", this should be fine.